### ISSUE TITLE
would NVIDA T400 or T600 work?

### ISSUE THREAD

**Issue by pete1019** [DualCoder/vgpu_unlock#87]
Would NVIDA T400 or T600 work?

Thanks

**Comment by polloloco** [comment]
Yes probably. I got it working with a T1000 on PVE 7.1. The 460 driver would cause constant VM freezes for me with that card so I'm using 450 now. I put my instructions in a git repo [here](https://gitlab.com/polloloco/vgpu-5.15), maybe that helps you

**Comment by kakosmakos** [comment]
> Would NVIDA T400 or T600 work?
> 
> Thanks

Hello. were you able to run vgpu unlock on a quadro t600 card?
